netebpfext/net_ebpf_ext.c

```diff
@@ -606,34 +609,33 @@ net_ebpf_extension_initialize_wfp_components(_Inout_ void* device_object)
     session.flags = FWPM_SESSION_FLAG_DYNAMIC;
 
     status = FwpmEngineOpen(NULL, RPC_C_AUTHN_WINNT, NULL, &session, &_fwp_engine_handle);
-    if (!NT_SUCCESS(status)) {
-        NET_EBPF_EXT_LOG_NTSTATUS_API_FAILURE(NET_EBPF_EXT_TRACELOG_KEYWORD_ERROR, "FwpmEngineOpen", status);
-        goto Exit;
-    }
+    NET_EBPF_EXT_BAIL_ON_API_FAILURE_STATUS("FwpmEngineOpen", status);
```

Comment:
```suggestion
    NET_EBPF_EXT_BAIL_ON_API_FAILURE_STATUS(FWPM_ENGINE_OPEN, status);
```

Reply:
Resolving this and other similar comments based on the first comment on this.